docs/react/guides/adding-assets.md

```diff
@@ -0,0 +1,39 @@
+# Adding Images, Fonts, and Files
+
+With Nx, you can **`import` assets directly from your TypeScript/JavaScript code**.
+
+```typescript jsx
+import React from 'react';
+import logo from './logo.png';
+
+const Header = () => <img src={logo} alt="Logo" />;
+
+export default Header;
+```
+
+This import will be replaced by a string of the image path when the application builds. To reduce the number of network requests, if the image file size is less than 10 kB, then the image will be inlined using [data URI](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs) instead of a path.
+
+This works in CSS files as well.
+
+```css
+.logo {
+  background-image: url(./logo.png);
+}
+```
+
+## Adding SVGs
+
+SVG images can be imported using the method described above.
+
+Alternatively, you can import SVG images as React components.
+
+```typescript jsx
+import React from 'react';
+import { ReactComponent as Logo } from './logo.svg';
+
+const Header = () => <Logo title="Logo" />;
```

Comment:
I'd explicitly showcase the SVG possibilities in that `#Adding SVGs guide` as it can get lost within the context...

I was scratching my head for a while how to use `import svgLogo from './logo.svg';`, although it's mentioned right there...

I suggest:

```ts
import { ReactComponent as Logo } from './logo.svg';
import svgLogo  from './logo.svg';

const Header = () => {
  return <div>
    <Logo title="Logo" />
    <img src="svgLogo />
</div>;
}
```

Lemme know WDYT and I'll send PR. thx!